tests/e2e/framework/common.go

```diff
@@ -62,14 +90,52 @@ func (f *Framework) cleanUpFromYAMLFile(p *string) error {
 	}
 
 	for _, d := range documents {
-		obj := &unstructured.Unstructured{}
-		if err := obj.UnmarshalJSON(d); err != nil {
-			return fmt.Errorf("failed to unmarshal object spec: %w", err)
+		obj, err := unmarshalJSON(d)
+		if err != nil {
+			return err
 		}
 		obj.SetNamespace(f.OperatorNamespace)
-		log.Printf("deleting %s %s", obj.GetKind(), obj.GetName())
+		log.Printf("deleting %s %s", obj.GetObjectKind().GroupVersionKind().Kind, obj.GetName())
 		if err := f.Client.Delete(context.TODO(), obj); err != nil {
-			return fmt.Errorf("failed to delete %s: %w", obj, err)
+			return fmt.Errorf("failed to delete %s: %w", obj.GetObjectKind().GroupVersionKind().Kind, err)
+		}
+	}
+	return nil
+}
+
+func (f *Framework) cleanUpProfileBundle(p string) error {
+	pb := &compv1alpha1.ProfileBundle{}
```

Comment:
Wondering if we can just delete without first get the object:

	pb := &compv1alpha1.ProfileBundle{
		ObjectMeta: metav1.ObjectMeta{
			Namespace: f.OperatorNamespace,
			Name:      p,
		},
	}
	err := f.Client.Delete(context.TODO(), pb)

Reply:
Yeah - that's probably reasonable.